extension/src/popup/views/SignTransaction.tsx

```diff
@@ -114,21 +119,46 @@ const SubmitButtonEl = styled(SubmitButton)`
   width: 12.43rem;
 `;
 
+const NetworkMismatchWarning = () => (
+  <>
+    <WarningMessage subheader={`Freighter is currently on ${NETWORK_NAME}`}>
+      <p>The transaction you're trying to sign is on {OTHER_NETWORK_NAME}.</p>
```

Comment:
You used a single quote in `you're` where you probably meant to use an apostrophe `you’re`. See https://github.com/stellar/product-conventions/blob/master/STYLEGUIDE.md#use-the-right-symbols.